mrp_production_batch/wizard/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import wizard_mrp_production_batch
+from . import wizard_mrp_production_batch_line
```

Comment:
Check OCA guidelines if this is the way to name wizards and also check the wizard filenames